lib/math_figures.py

```diff
@@ -0,0 +1,6 @@
+def area(_n: float) -> float:
```

Comment:
The function should be named precisely. It is not just an area - it is exactly `square_area`